### PR TITLE
Basic functionality

### PR DESCRIPTION
**Pull request by jh62**
- Created the Main.class
- Created the GameScreen.class and the first screen of the game (ScreenMainMenu.class)
- Added basic cursor movement for the menu.

### CODE REVIEW

File: core/src/com/game87/game/main/Main.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.game87.game.screen.*;
+
+/**
+ *
+ * @author Pablo Leite
+ */
+public class Main extends Game {
+
+    public static final Game INSTANCE = new Main();
```
Comment (3vilStar) [inline core/src/com/game87/game/main/Main.java]: Perhaps use Gdx.app.getApplicationListener()?

Comment (jh62) [inline core/src/com/game87/game/main/Main.java]: Game implements application listener. It already has everything you need. There's no real benefit.

Comment (jh62) [inline core/src/com/game87/game/main/Main.java]: Should I commit the code to the master branch?

**Comment by gabe00122** [comment]
Looks good as a start, you should commit.

**Pull request merged**